qa/L0_perf_resnet/run_test.sh

```diff
@@ -73,20 +66,58 @@ fi
 
 set +e
 
-# Run the model once to warm up. Some frameworks do optimization on the first requests.
-# Must warmup similar to actual run so that all instances are ready
-$PERF_CLIENT -v -i ${PERF_CLIENT_PROTOCOL} -m $MODEL_NAME -p${MEASUREMENT_WINDOW} \
-                -b${STATIC_BATCH} --concurrency-range ${CONCURRENCY}
+# Overload use of PERF_CLIENT_PROTOCOL for convenience with existing test and 
+# reporting structure, though "triton_c_api" is not strictly a "protocol".
+# NOTE: tritonserver will be ran in-process with perf_analyzer
+if [[ "${PERF_CLIENT_PROTOCOL}" == "triton_c_api" ]]; then
+    # Using C API requires extra info to start in-process server
+    # NOTE: PA C API doesn't expose --backend-directory, but defaults to
+    #       ${TRITON_DIR}/backends
+    # NOTE: PA C API doesn't expose --backend-config, but this is only used
```

Comment:
This comment is confusing. I don't think we need --backend-config tfserving grpc test.

Reply:
Shouldn't all tfserving benchmarking be in L0_perf_tfs?

Reply:
[This](https://github.com/triton-inference-server/server/blob/2f60b85e37ed6578cb25623013364bebba104341/qa/L0_perf_resnet/test.sh#L212-L230) was what I was referring to, but I agree I have too many comments here. Cleaning them up, and removing mention of this.